src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+}
+
+li,
+p,
+ul,
+body {
+  list-style: none;
```

Comment:
You don't need this rule for every these tags but just for `ul`. So create separate selector for `ul` with this rule.

Reply:
Thank you!